pom.xml

```diff
@@ -41,6 +44,7 @@
       <groupId>org.testng</groupId>
       <artifactId>testng</artifactId>
       <version>7.9.0</version>
+     <version>7.9.0</version>
```

Comment:
<div>
<h3>Code Review Agent Run <a href = https://github.com/Amruta101998/selenium-automation/pull/2/files#2122405823>#64877</a> - 05/21/2024, 04:52 pm</h3>

<p>🔴 High importance</p>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Duplicate version tag for the same dependency (org.testng:testng) introduces ambiguity and potential conflicts in dependency resolution which can lead to unexpected behavior or runtime errors.
</div>
<div id="fix">
<b>Fix:</b> Remove the duplicate version tag to ensure clear and consistent dependency resolution.
</div>
<details>
<summary>
Code suggestion
</summary>
<div>
<pre><code>
 @@ -44,7 +44,6 @@
      <artifactId>testng</artifactId>
      <version>7.9.0</version>
 -    <version>7.9.0</version>
      <scope>test</scope>
 </dependency>
</code></pre>
</div>
</details>
<h4><a href="https://github.com/Amruta101998/selenium-automation/pull/2/files#2122405823"><-- Code Review Agent Run#64877</a></h4>
<i>AI Code Review powered by <img src="https://bito.ai/wp-content/uploads/2023/10/Logo-Bito-Black-cropped.svg" alt="Bito Logo" width="50" height="25"></i>
</div>
<br>

<br>

</div>